rollup.config.js

```diff
@@ -0,0 +1,252 @@
+import commonjs from 'rollup-plugin-commonjs';
+import babel from 'rollup-plugin-babel';
+import json from 'rollup-plugin-json';
+import autoExternal from 'rollup-plugin-auto-external';
+import cleanup from 'rollup-plugin-cleanup';
+import nodeGlobals from 'rollup-plugin-node-globals';
+import builtins from 'rollup-plugin-node-builtins';
```

Comment:
This plugin doesn't work very well with `crypto`. Are we sure this isn't bringing any problem?

Reply:
This plugin does the browser e2e tests wouldn't success otherwise. ``builtins`` does include the ``crypto-browserify`` shim as expected.

Reply:
I'm resolving this. There's another conversation about this.